src/styles/blocks/content.scss

```diff
@@ -0,0 +1,8 @@
+.content {
+  display: flex;
+  flex-direction: column;
+
+  &--margin_auto {
+    margin: 0 auto;
+  }
```

Comment:
Don't add margins to BEM block even through the modifier. Use BEM mix for those.

Reply:
Fixed.